src/gui/preferences.c

```diff
@@ -392,6 +400,28 @@ static void init_tab_general(GtkWidget *dialog, GtkWidget *stack, dt_gui_themetw
   g_signal_connect(G_OBJECT(widget), "changed", G_CALLBACK(theme_callback), 0);
   gtk_widget_set_tooltip_text(widget, _("set the theme for the user interface"));
 
+  label = gtk_label_new(_("display"));
+  gtk_widget_set_halign(label, GTK_ALIGN_START);
+  widget = gtk_combo_box_text_new();
+  labelev = gtk_event_box_new();
+  gtk_widget_add_events(labelev, GDK_BUTTON_PRESS_MASK);
+  gtk_container_add(GTK_CONTAINER(labelev), label);
+  gtk_grid_attach(GTK_GRID(grid), labelev, 0, line++, 1, 1);
+  gtk_grid_attach_next_to(GTK_GRID(grid), widget, labelev, GTK_POS_RIGHT, 1, 1);
+
+  gtk_combo_box_text_append_text(GTK_COMBO_BOX_TEXT(widget), "default");
+  gtk_combo_box_text_append_text(GTK_COMBO_BOX_TEXT(widget), "standard monitor");
+  gtk_combo_box_text_append_text(GTK_COMBO_BOX_TEXT(widget), "standard monitor, (slow)");
```

Comment:
I'd say "slow computers" or "fast(er) ".
"slow" literal is used in other places to say that it's slower with that option enabled (e.g. demosaic modes).

Reply:
+1

Reply:
ok. Fixed by introducing just a single button